vnext/src/Libraries/Alert/Alert.windows.js

```diff
@@ -52,6 +68,19 @@ export class Alert {
       }
     });
   }
+
+  static prompt(
```

Comment:
>prompt [](start = 9, length = 6)

Looks like facebook's docs are wrong (they say only AlertiOS has prompt), and this explains how it got missed when Hunter implemented Alert during his internship.  

I'm happy to file a doc bug, where are these being opened?

Reply:
Took a closer look and now see the API is on Alert but is only implemented on iOS.  We could certainly implement it on Windows but it's not very useful without an Android implementation too.  If Alert is going to stay in lean core, I think that's what's needed (both Android and Windows implementations).

---
In reply to: [355615926](https://github.com/microsoft/react-native-windows/pull/3730#discussion_r355615926) [](ancestors = 355615926)